_docs-sources/iac/whats-this/services.md

```diff
@@ -1,11 +1,35 @@
 # What is a Service?
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Eu nisl nunc mi ipsum faucibus vitae aliquet. Cras adipiscing enim eu turpis egestas pretium aenean pharetra magna. Nam at lectus urna duis convallis convallis. Eget est lorem ipsum dolor sit amet consectetur adipiscing elit. Interdum varius sit amet mattis vulputate enim. Massa id neque aliquam vestibulum. Nunc sed augue lacus viverra vitae congue eu consequat ac. Volutpat ac tincidunt vitae semper. Placerat orci nulla pellentesque dignissim enim. Dui sapien eget mi proin sed libero enim sed. Sem viverra aliquet eget sit. Dis parturient montes nascetur ridiculus. Quam id leo in vitae. Tortor dignissim convallis aenean et tortor at risus viverra adipiscing. Elit eget gravida cum sociis natoque penatibus et.
+The Gruntwork Service Catalog consists of a number of reusable, customizable, battle-tested, production-grade infrastructure-as-code services that you can use to deploy and manage your infrastructure, including Docker orchestration, EC2 orchestration, load balancing, networking, databases, caches, monitoring, alerting, CI/CD, secrets management, VPN, and much more.
```

Comment:
Last one then I'll stop - we haven't defined battle-tested, and it's unclear if that means our modules have been proven out at high scale or if they've been successfully deployed across many different use cases and companies, etc.